server/controllers/reviews.js

```diff
@@ -2,66 +2,78 @@ import models from '../models';
  */
-export const postReview = (req, res) => {
-  const userId = req.userId;
-  const recipeId = req.params.recipeId;
-  const content = (req.body.content || '').replace(/\s+/g, ' ');
+export default class Review {
+  /**
+   * Post a review on a recipe
+   *
+   * @param {object} req - HTTP Request
+   * @param {object} res - HTTP Response
+   * @returns {object} Class instance
+   * @memberof Review
+   */
+  postReview(req, res) {
+    const userId = req.userId;
+    const recipeId = req.params.recipeId;
+    const content = (req.body.content || '').replace(/\s+/g, ' ');
 
-  const validateReviewContentError = validate.validateReviewContent(content);
-  if (validateReviewContentError) {
-    return res.status(403).json({
-      success: false,
-      message: validateReviewContentError });
-  }
+    const validateReviewContentError = validate.validateReviewContent(content);
+    if (validateReviewContentError) {
+      return res.status(403).json({
+        success: false,
+        message: validateReviewContentError });
+    }
+
+    review
+      .create({
+        content,
+        userId,
+        recipeId
+      })
+      .then((createdReview) => {
+        res.status(201).json({
+          success: true,
+          createdReview });
+      })
+      .catch(() => res.status(503).json({
+        success: false,
+        message: 'Error Posting Review' }));
 
-  const newReview = review
-    .create({
-      content,
-      userId,
-      recipeId
-    })
-    .then((createdReview) => {
-      res.status(201).json({
-        success: true,
-        createdReview });
-    })
-    .catch(() => res.status(503).json({
-      success: false,
-      message: 'Error Posting Review' }));
+    return this;
+  }
 
-  return newReview;
-};
+  /**
+   * Get a list of reviews on a recipe
+   *
+   * @param {object} req - HTTP Request
+   * @param {object} res - HTTP Response
+   * @returns {object} Class instance
+   * @memberof Review
+   */
+  getReviews(req, res) {
+    const recipeId = req.params.recipeId;
 
-/** Get a list of reviews on a recipe
- * @exports getReviews
- * @param  {object} req - request
- * @param  {object} res - response
- * @return {object} The status/reviews fetched
- */
-export const getReviews = (req, res) => {
-  const recipeId = req.params.recipeId;
-  const newReview = review
-    .findAll({
-      where: { recipeId },
-      include: [
-        { model: models.User, attributes: ['name', 'updatedAt'] }
-      ]
-    })
-    .then((reviews) => {
-      res.status(201).json({
-        success: true,
-        data: reviews });
-    })
-    .catch(() => res.status(503).json({
-      success: false,
-      message: 'Error Fetching Reviews' }));
+    review
+      .findAll({
+        where: { recipeId },
+        include: [
+          { model: models.User, attributes: ['name', 'updatedAt'] }
+        ]
+      })
+      .then((reviews) => {
+        res.status(201).json({
+          success: true,
+          data: reviews });
```

Comment:
adding a message to your response would be a good idea

Reply:
okay, I will work on that too